force-app/main/domain/Households.cls

```diff
@@ -137,12 +137,8 @@ public inherited sharing class Households extends fflib_SObjects2 {
         // the Account to include mapping of the Primary Address Type.
         //Deactivate the Trigger that syncs Addresses to Accounts anc Contacts,
         // since the Household Account and Contact are already configured correctly.
-        TDTM_ProcessControl.toggleTriggerState('Address__c', 'ADDR_Addresses_TDTM',
```

Comment:
Removing these toggles is great! Let's update/remove the comment above to reflect the change.

Reply:
Let's make sure the trigger toggle removal will not introduce some unnecessary SOQL/DML update. We had the issue inserting/deleting blank address.